source/reference/compatibility.txt

```diff
@@ -128,6 +128,37 @@ Write Preference Behavior
 legacy :binary:`mongo` shell. To disable retryable writes, use 
 :option:`--retryWrites=false <mongosh --retryWrites>`.
 
+
+ObjectId Methods and Attributes
+-------------------------------
+
+The :ref:`ObjectId() <server-objectid>` methods work differently in
+``mongosh`` than in the legacy ``mongo`` shell.
+
+.. list-table::
```

Comment:
Just curious if you think having two separate tables might help here, one for the explanations like you had before and one for the examples. A separate table for the examples might be useful for a clearer side-by-side comparison of the outputs. As in something like:

```
The following table compares shell outputs using an example
ObjectId ``ObjectId(6419ccfce40afaf9317567b7)``:

.. list-table::

   * - Method or Attribute
     - ``mongo`` Output
     - ``mongosh`` Output

   * - ``ObjectId.str``
      - ``6419ccfce40afaf9317567b7``
      - Not available.

   * - ``ObjectId.valueOf()``
     - 6419ccfce40afaf9317567b7
     - ObjectId("6419ccfce40afaf9317567b7")
```
 ... and so on.